Add config file examples to sensuctl reference. (#1205)
sensu/sensu-docs: content/sensu-go/5.2/sensuctl/reference.md

- [Managing resources](#managing-resources)
- [Time formats](#time-formats)
- [Shell auto-completion](#shell-auto-completion)

> - [Config files](#)

Reviewer: Should this have more after the `#`? Maybe `configuration-files`?
```suggestion
- [Config files](#configuration-files)
```

Author: Right, I hadn't determined what the link would be when I was writing. Will update with this.

create  delete  import  list
{{< /highlight >}}

## Configuration files

> When configuring sensuctl, configuration files will be created that contain information for connecting to your Sensu Go deployment. You can find them at `$HOME/.config/sensu/sensuctl/profile` and `$HOME/.config/sensu/sensuctl/profile`. Here are two examples:

Reviewer 2: ```suggestion
During configuration, sensuctl creates configuration files that contain information for connecting to your Sensu Go deployment. You can find them at `$HOME/.config/sensu/sensuctl/profile` and `$HOME/.config/sensu/sensuctl/profile`. For example:
```
